protocol-units/settlement/mcr/client/src/eth_stub.rs

```diff
@@ -0,0 +1,247 @@
+use crate::{AcceptedBlockCommitment, CommitmentStream, McrSettlementClientOperations};
```

Comment:
Let's not call this `eth_stub` unless we're using a stub contract (which is a good idea for separating testing concerns).

Reply:
Yes, but I've no idea that seems to me good. That why I use `eth_strb`. If you can propose something.

Reply:
I call it `eth_client.rs` and move all eth Tx send code in `send_eth_tx.rs `file.